src/flask/routes/chef.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+from flask import Blueprint, request
+from injector import Injector
+
+from src.api.controllers.chef_controller import ChefController
+from src.constants.audit import ExternalUsers
+from src.core.di_config import DiProviders
+from src.flask.schemas.chef_schema import ChefSchema
+from src.lib.entities.sqlalchemy_orm_mapping import Chef
+
+chef_blueprint = Blueprint("chef_routes", __name__)
+
+ioc = Injector(DiProviders)
+chef_controller = ioc.get(ChefController)
+chef_schema = ChefSchema()
+chef_schemas = ChefSchema(many=True)
+
+
+@chef_blueprint.route("/chefs", methods=["POST"])
+def add():
+    json_chef_data = request.get_json()
+    chef = chef_schema.load(json_chef_data)
+    chef_controller.add(chef)
+
+    return chef_schema.dump(chef)
+
+
+@chef_blueprint.route("/chefs/<chef_id>", methods=["GET"])
+def get_by_id(chef_id):
+    chef = chef_controller.get_by_id(chef_id)
+    return chef_schema.dump(chef)
+
+
+@chef_blueprint.route("/chefs", methods=["GET"])
+def get_all():
+    chefs = chef_controller.get_all()
+    return chef_schemas.dump(chefs)
+
+
+@chef_blueprint.route("/chefs/<chef_id>", methods=["PUT"])
+def update_chef_by_id(chef_id):
+    json_chef_data = request.get_json()
+    chef = chef_schema.load(json_chef_data)
+    chef.updated_by = ExternalUsers.API_USER.value
```

Comment:
Set the user through the client `postman` for now, do not harcdode it

Reply:
done.